tests/README.md

```diff
@@ -0,0 +1,17 @@
+# Running the tests
+
+## Test drug categorization
+```
```

Comment:
As above, there is no need to specify each one individually, that's what `pytest` does